<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAuthor</class>
 <widget class="QMainWindow" name="AddAuthor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить автора</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Icons.qrc">
    <normaloff>:/Icons/MainIcon/Resources/MainIcons/free-icon-book-4341050.png</normaloff>:/Icons/MainIcon/Resources/MainIcons/free-icon-book-4341050.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="surname_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Отчество</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ptr_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="add_author">
      <property name="text">
       <string>Добавить автора</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
